README.md

```diff
@@ -18,12 +20,53 @@ Some layers include the corresponding OTel language SDK for the Lambda. This all
 * **Is the Lambda layer provided or do I need to build it and distribute it myself?**
     > This repository does not provide pre-build Lambda layers. They must be built manually and saved in your AWS account. This repo has files to facilitate doing that. More information is provided in [the Collector folder's README](collector/README.md).
 
 ## Design Proposal
+
 To get a better understanding of the proposed design for the OpenTelemetry Lamda extension, you can see the [Design Proposal here.](docs/design_proposal.md)
+
+## Features
+
+The following is a list of features provided by the OpenTelemetry layers.
+
+### Custom Context Propagation
+
+Context can be propagated through various mechanisms (e.g. http headers (APIGW), message attributes (SQS), ...). In some cases, it may be required to pass a custom context propagation extractor in lambda through configuration, this feature allows this through Lambda instrumentation configuration.
+
+### Disable AWS Context propagation via environment variable
+
+The context propagation when Active Tracing is enabled uses the `_X_AMZN_TRACE_ID` environment variable present in the Lambda runtime. To disable this behaviour, implementations have added the `OTEL_LAMBDA_DISABLE_AWS_CONTEXT_PROPAGATION` environment variable which can be set to `true`.
+
+### Semantic conventions
+
+The Lambda language implementation follows the semantic conventions specified in the OpenTelemetry Specification.
+
+### Support matrix
+
+The table below captures the state of various features and their levels of support different runtimes.
+
+| Runtime       | Custom Context Propagation | Disable AWS Propagation | Semantic conventions |
+| ------------- | :------------------------: | :---------------------: | :------------------: |
+| Node.js 16    |                            |                         |                      |
+| Node.js 14    |                            |                         |                      |
+| Python 3.9    |                            |                         |                      |
+| Python 3.8    |                            |                         |                      |
+| Python 3.7    |                            |                         |                      |
+| Java 11       |                            |                         |                      |
+| Java 8        |                            |                         |                      |
+| .NET Core 3.1 |                            |                         |                      |
+| .NET 6        |                            |                         |                      |
+| .NET 5        |                            |                         |                      |
+| Go 1.x        |                            |                         |                      |
+| Ruby 2.7      |                            |                         |                      |
```

Comment:
Are there actually differences in whether they support each of these things between the version of the languages?

I think breaking out the semantic conventions to at least `Trace General`, `Trace Incoming`, `Trace Outgoing` and `Metrics` and transposing the matrix so the languages are the headers would be better -- partly assuming languages can be combined so there aren't 11 columns.

Reply:
I don't know if there *should* be differences between the language versions, I want to treat this matrix as a bit of a  test plan, to ensure we've at least confirmed this works in all the supported languages. I worry that otherwise we would say "X works in node.js 16 therefore will work in all node.js, we won't test it", only to find out there is a slight difference in some library's implementation or dependency causing X not to work

Reply:
We could always combine the languages w/ the understanding that something wouldn't be marked as "supported" in any language until its been tested in all supported runtimes?

Reply:
I'd support that (combine them and only mark it if all are supported). Or could mark with a different character, plus have a footnote if 1 version has a specific reason it will never support this.